server/database/seeders/20190309084305-admin-user.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  up(queryInterface) {
+    return queryInterface.bulkInsert('Users', [{
+      firstname: 'admin',
+      lastname: 'admin',
+      username: 'admin',
+      email: 'ayooluwabayo@gmail.com',
+      password: '$2a$08$G/5b9Oq3rla2pjb/imcZ8u.m0EQlbKfG.gwGJG6iOfPVm9rZqNVmC',
+      verifiedEmail: true,
+      isAdmin: true,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      firstname: 'ayo',
+      lastname: 'ayo',
+      username: 'ayo',
+      email: 'ayo-oluwa.adebayo@andela.com',
+      password: '$2a$08$G/5b9Oq3rla2pjb/imcZ8u.m0EQlbKfG.gwGJG6iOfPVm9rZqNVmC',
+      verifiedEmail: true,
+      isAdmin: false,
+      createdAt: new Date(),
+      updatedAt: new Date()
+    },
+    {
+      firstname: 'bayo',
+      lastname: 'bayo',
+      username: 'bayo',
+      email: 'uniqueayo@yahoo.com',
+      password: '$2a$08$G/5b9Oq3rla2pjb/imcZ8u.m0EQlbKfG.gwGJG6iOfPVm9rZqNVmC',
```

Comment:
Is there a reason why you need to add the hashed password here directly? kindly confirm this.

Reply:
You could call the hash function then passing the password; What do you think?

Reply:
I needed the hashed password to log in and run my tests. However, it is more efficient to call the hash function as pointed out.
Thank you for pointing this out. I'll do the needful.